hawc/apps/summary/models.py

```diff
@@ -247,6 +252,10 @@ def to_docx(self, base_url: str = ""):
         return ReportExport(docx=docx, filename=self.slug)
 
     def clean(self):
+        # assessment id should match content
```

Comment:
This is my idea on how to make sure `content` doesn't include any objects that are outside of the scope of this assessment; if `assessment_id` is in content, it MUST match the `SummaryTable`'s assessment_id. All other validation (aka study/metric validation for study outcome tables) are done in the table's pydantic model using the given `assessment_id`.

Reply:
I don't think a user should be able to provide this in the content; we should update any method calls to pass the assessment_id and the content on the backend so that it's immutable and available where needed. Is this a big lift?